<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deposit_calc</class>
 <widget class="QDialog" name="deposit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deposit Calc</string>
  </property>
  <widget class="QLabel" name="deposit_sum_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_sum">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_sum_label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit period (1 to 600 months)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_term">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="deposit_percent">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_sum_label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Interest rate (from 0 to 100)</string>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_sum_label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Capitalization of interest</string>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_sum_label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Payment frequence</string>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_sum_label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Accrued interest</string>
   </property>
  </widget>
  <widget class="QLabel" name="interest_income">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>660</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_sum_label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>710</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="tax_sum">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>710</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_sum_label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>760</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit total at term end</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_deposit_sum">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>760</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Count</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>123</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioB_end_cap">
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioB_mon_cap">
      <property name="text">
       <string>Monthly</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>123</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioB_mon_pay">
      <property name="text">
       <string>Monthly</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioB_end_pay">
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <!-- <width>211</width> -->
     <width>271</width>
     <height>231</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>+/-</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Added as %</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total</string>
    </property>
   </column>
  </widget>
  <widget class="QCheckBox" name="checkBox_add">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
   <string>Monthly info/deposit-withdrawal</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tax_percent">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="deposit_sum_label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tax rate (from 0 to 100)</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>600</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Positive value - deposit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Negative value - withdrawal</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
